spec/features/deleting_a_user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.feature 'Finding a user' do
+  scenario 'successfully' do
+    sign_in_as_admin
+    @user = FactoryBot.create(:user)
+    stub_request(:post, 'https://testdomain/oauth/token')
```

Comment:
I think things would be clearer and easier to follow if these stubbed Auth0 requests were abstracted out into well-named methods. I got a bit confused when I saw `"https://testdomain/api/v2/users/#{@user.auth_id}"` as it looks very similar to one of our own API endpoints in the app and it wasn't clear that `testdomain` is the stubbed domain for Auth0.